src/seedu/addressbook/commands/Command.java

```diff
@@ -40,13 +40,21 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
      * Executes the command and returns the result.
      */
     public CommandResult execute(){
-        throw new UnsupportedOperationException("This method should be implement in child classes");
+        throw new UnsupportedOperationException("This method should be implemented in child classes.");
```

Comment:
Could this be potentially changed to become an abstract method as well?

Reply:
Yes, that's a good idea.